azurerm/internal/services/storage/tests/resource_arm_storage_account_test.go

```diff
@@ -819,6 +819,25 @@ func TestAccAzureRMStorageAccount_largeFileShare(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMStorageAccount_withPremiumBlockBlobStorageAndEnabledHns(t *testing.T) {
```

Comment:
Could we update this to the new test framework?
```suggestion
func TestAccStorageAccount_hnsWithPremiumStorage(t *testing.T) {
```

Reply:
updated